docs/source/operators/deploy-kubernetes.md

```diff
@@ -54,48 +54,48 @@ helm install --namespace enterprise-gateway enterprise-gateway https://github.co
-| `kip.image`                    | Kernel Image Puller image name and tag to use. Ensure the tag is updated to the version of the Enterprise Gateway release you wish to run.                                                                                                       | `elyra/kernel-image-puller:VERSION`, where `VERSION` is the release being used |
-| `kip.imagePullPolicy`          | Kernel Image Puller image pull policy. Use `IfNotPresent` policy so that dev-based systems don't automatically update. This provides more control. Since formal tags will be release-specific this policy should be sufficient for them as well. | `IfNotPresent`                                                                 |
-| `kip.interval`                 | The interval (in seconds) at which the Kernel Image Puller fetches kernelspecs to pull kernel images.                                                                                                                                            | `300`                                                                          |
-| `kip.pullPolicy`               | Determines whether the Kernel Image Puller will pull kernel images it has previously pulled (`Always`) or only those it hasn't yet pulled (`IfNotPresent`)                                                                                       | `IfNotPresent`                                                                 |
-| `kip.criSocket`                | The container runtime interface socket, use `/run/containerd/containerd.sock` for containerd installations                                                                                                                                       | `/var/run/docker.sock`                                                         |
-| `kip.defaultContainerRegistry` | Prefix to use if a registry is not already specified on image name (e.g., quay.io/elyra/kernel-py:2.6.0)                                                                                                                                         | `docker.io`                                                                    |
+| **Parameter**                         | **Description**                                                                                                                                                                                                                                  | **Default**                                                                    |
+| ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------ | --- | ---------- | ----------------- | ------- |
```

Comment:
I think we have two many columns here.
```suggestion
| ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------ | 
```

Reply:
looks like prettier made these changes to the table, i'll see if i can fix it